test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetListCommandTest.cs

```diff
@@ -902,8 +901,8 @@ public void ListCommand_InvalidInput_V3_NonExistent(string invalidInput)
                 "The run did not fail as desired. Simply got this output:" + result.Item2);
 
             Assert.True(
-                result.Item3.Contains($"Unable to load the service index for source {invalidInput}."),
-                "Expected error message not found in " + result.Item3
+                result.AllOutput.Contains($"Unable to load the service index for source '{invalidInput}'."),
```

Comment:
Why not just change this to an `Assert.Contains`, and then there's no need for the "Expected error" string.

Reply:
sure. fixed in next iteration.

Reply:
There's no functional difference there, but fluent assertions doesn't truncate the output like Assert.Contains would. 

That's the primary reason why some engs prefer it over the base xunit assertions.